src/components/node/sul-library/page-display.tsx

```diff
@@ -8,31 +8,30 @@ const SulLibrary = async ({node, ...props}: { node: Library }) => {
   node.su_library__paragraphs = await fetchComponents<StanfordParagraph>(node.su_library__paragraphs || []);
   node.su_library__paragraphs = node.su_library__paragraphs.filter(item => !!item?.id);
   const fullWidth = node.layout_selection?.resourceIdObjMeta?.drupal_internal__target_id === 'sul_library_full_width'
+  const containerClasses = ["mb-50", "lg:max-w-[980px]", "mx-auto", "centered"];
+  const containerStyles = containerClasses.join(' ');
 
   return (
     <article className="mb-50 @container" {...props}>
+      {node.su_library__hours &&
+        <div className={containerStyles}>
+          <LibraryAdditionalHours hoursId={node.su_library__hours}/>
+        </div>
+      }
+
+      {<ParagraphRows items={node.su_library__paragraphs} fullWidth={fullWidth}/>}
 
-      {(node.sul_library__a11y || node.su_library__hours) &&
-        <div
-          className="centered mb-50 flex flex-col @6xl:flex-row gap-[90px]">
-          {node.sul_library__a11y &&
-            <div className="order-last @6xl:order-first flex-1 basis-1/2">
+      {node.sul_library__a11y &&
+            <div className={containerStyles}>
               <div className="shadow-md py-20 px-30 w-fit mx-auto border border-black-10">
```

Comment:
if the container has max width and mx-auto, remove the w-fit here. or better yet, just reduce the extra divs and apply th necessary styles here.

![image](https://github.com/SU-SWS/sulgryphon-nextjs/assets/7185045/159dcea6-9998-4cbe-b604-9c8975173ad0)

Reply:
https://su-library-git-sul23-418-accessibilit-28ad7a-stanford-libraries.vercel.app/libraries/academy-hall-redwood-city-campus

Reply:
@pookmish Okay. Good to go.